src/System.Security.Cryptography.Cng/tests/project.lock.json

```diff
@@ -30,23 +33,26 @@
           "lib/DNXCore50/System.Console.dll": {}
         }
       },
-      "System.Diagnostics.Debug/4.0.10-beta-22703": {
+      "System.Diagnostics.Debug/4.0.10-beta-23127": {
```

Comment:
I double checked, all the beta upgrades in this lock file are bringing it in line with the rest of the modules.  This addition and the last upgrade must have crossed in the night.